server/mergin/sync/private_api_controller.py

```diff
@@ -305,26 +304,15 @@ def unsubscribe_project(id):  # pylint: disable=W0612
 
 @auth_required
 def update_project_access(id: str):
-    """Modify shared project access
+    """Modify the project's public flag
 
     :param id: Project uuid
     """
     project = require_project_by_uuid(id, ProjectPermissions.Update)
 
     if "public" in request.json:
```

Comment:
I think if it is the only parameter, let's make it required.

Reply:
Also we can change response type, and even change URI?

Reply:
Any hints on what to return and the new URI?

Reply:
/app/projects/{id}/public { public: boolean } ??

Reply:
-> none?